### ISSUE TITLE
The application “Visual Studio Code.app” can’t be opened.

### ISSUE THREAD

**Issue by epitaphmike** [Microsoft/vscode#51461]
While this may seem like a duplicate to https://github.com/Microsoft/vscode/issues/21230, I am having the same issue and it is not a company policy problem or Archive Utility. I was able to download and install perfectly on another machine.

- VSCode Version: 1.24
- OS Version: OSX 10.11.6

Steps to Reproduce:

1. Downloaded latest version of VS Code

2. Replaced App in Applications folder

3. Tried opening the App and got this message
<img width="457" alt="screen shot 2018-06-08 at 10 53 02 am" src="https://user-images.githubusercontent.com/677832/41164849-245f730e-6b0a-11e8-8408-dd43338812da.png">

4. I have restarted my machine

5. re-downloaded

6. Deleted the App from Applications

7. gone through steps to remove CODE from Application Support, as well as any other files I could find like .vscode
8. re-downloaded and tried the Insiders Build.

__Console Output__
```
6/8/18 10:56:24.067 AM Finder[1011]: spawn_via_launchd() failed, errno=111 label=[0x0-0x55055].com.microsoft.VSCode path=/Users/<username>/Desktop/Visual Studio Code.app/Contents/MacOS/Electron flags=1 : LaunchApplicationClient.cp #963 LaunchApplicationWithSpawnViaLaunchD() q=com.apple.root.default-qos

6/8/18 10:56:24.067 AM Finder[1011]: spawn_via_launchd() failed, errno=111 label=[0x0-0x55055].com.microsoft.VSCode path=/Users/<username>/Desktop/Visual Studio Code.app/Contents/MacOS/Electron flags=1
```

**Comment by joaomoreno** [comment]
If you run it directly in the Terminal, what's the full output? E.g:

```
/Users/<username>/Desktop/Visual Studio Code.app/Contents/MacOS/Electron
```

**Comment by joaomoreno** [comment]
Did you try running it in the Terminal?

**Comment by epitaphmike** [comment]
I doesn't do anything. I get the popup and the command finishes.
<img width="708" alt="screen shot 2018-06-13 at 10 06 57 am" src="https://user-images.githubusercontent.com/677832/41356520-17643e82-6ef2-11e8-9fe8-0a20273f1b42.png">

**Comment by epitaphmike** [comment]
Ok. I deleted the app. Downloaded it again, for the nth time, opened it from the terminal (iTerm2) and it worked....

<img width="696" alt="screen shot 2018-06-13 at 10 15 23 am" src="https://user-images.githubusercontent.com/677832/41356930-fb6c3dd2-6ef2-11e8-8fc3-46e9762ea3d4.png">

I clicked "Open" and a new terminal (Terminal) window opened along with the Application.

<img width="908" alt="screen shot 2018-06-13 at 10 16 30 am" src="https://user-images.githubusercontent.com/677832/41357006-33f18504-6ef3-11e8-8cc7-aaca1ede5faf.png">

Not sure why it worked this time, but it did. Hopefully this helps someone else out in the future.

Thanks for your assistance.
